src/Panel/Model.php

```diff
@@ -139,21 +139,31 @@ public function image($settings = null): ?array
                 ];
 
                 // for lists
-                $settings['list'] = [
-                    'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
-                    'srcset' => $image->srcset([
-                        '1x' => [
-                            'width' => 38,
-                            'height' => 38,
-                            'crop' => 'center'
-                        ],
-                        '2x' => [
-                            'width' => 76,
-                            'height' => 76,
-                            'crop' => 'center'
-                        ],
-                    ])
-                ];
+                if (($settings['cover'] ?? false) === false) {
+                    $settings['list'] = [
+                        'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
+                        'srcset' => $image->srcset([
+                            38,
+                            76
+                        ])
+                    ];
+                } else {
+                    $settings['list'] = [
+                        'url' => 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw',
```

Comment:
I know it's a minor detail, but can we store the placeholder gif in a variable to not repeat it?

Reply:
@bastianallgeier You're right. How about solving it with new `->panel()->imagePlaceholder()` method for all usages?

Reply:
I like that! @distantnative what do you think?

Reply:
Do we need a method for that? Wouldn't a simple variable be fine?

Reply:
Ah seeing it pops up in various files. Ok sure, but I think we could make it a protected method to not clutter the reference

Reply:
If it is `protected`, we will not be able to access it from models. So let's just continue with variables, as Bastian said.